changelog.d/20240206_151901_fahad.khalid.md

```diff
@@ -0,0 +1 @@
+[Improvement] Remove base_url from GRADES_DOWNLOAD. (by @FahadKhalid210)
```

Comment:
This is actually a bugfix; and we should not describe _what_ we are doing in this commit, but _why_. The "what" is removing the `base_url` entry. The "why" is to prevent a 500 error caused by an ImproperlyConfigured exception when downloading profile information as CSV, and when the minio plugin is enabled.

So I would rewrite this changelog entry as follows:

```
- [Bugfix] When the minio plugin is enabled (particularly on Kubernetes), fix a 500 error on downloading profile information as CSV. (by @FahadKhalid210)
```

Reply:
If we run tutor without `minio` and this fix, it works fine.
But if we remove base_url and disable `minio` and click on `Download Profile as a CSV` then its available in `Reports Available for Download ` but URL is not correct(missing grades in url) and got 404.

<img width="1066" alt="image" src="https://github.com/overhangio/tutor/assets/79192414/81119994-43ae-4048-8fea-c4400a526016">

<img width="1583" alt="image" src="https://github.com/overhangio/tutor/assets/79192414/2c0e5c61-845c-42b2-bf5c-71ec0a66da34">

Reply:
Since this fix pertains to the minio plugin, it would be more suitable to address it there. However, let me investigate if we can fix this url issue(404) in Tutor core.

Reply:
@regisb URL issue is fixed by updating the `location`. 
I have tested this fix with and without minio plugin and it works fine now.
URL without minio plugin:
`http://local.overhang.io:8000/media/bb6d638a296059742509a0d319ddf8456b6dbf9a/edX_DemoX_Demo_Course_student_profile_info_2024-02-09-1101.csv`

URL with minio plugin:
`http://files.local.overhang.io:9000/openedx/openedx/media/bb6d638a296059742509a0d319ddf8456b6dbf9a/edX_DemoX_Demo_Course_student_profile_info_2024-02-09-1104.csv`